config.yml

```diff
@@ -55,5 +168,5 @@ Str15: "AFKPGC version: %s, the plugin is %s"
 Str16: "AFKPGC is alive"

```

Comment:
These can be removed, they were used by the old CommandHandler and the Message class, which was completly removed